mongodbatlas/resource_mongodbatlas_database_user_test.go

```diff
@@ -329,3 +383,41 @@ func testAccMongoDBAtlasDatabaseUserWithLabelsConfig(projectID, roleName, userna
 		}
 	`, projectID, roleName, username, labelsConf)
 }
+
+func testAccMongoDBAtlasDatabaseUserWithRoles(username, password, projectID string, rolesArr []*matlas.Role) string {
+
```

Comment:
Delete the unnecesary NL at the beginning, but the rest LGTM

Reply:
thanks